apps/01PigLatin.js

```diff
@@ -8,31 +8,43 @@ var rl = readline.createInterface({
 });
 
 function pigLatin(word) {
+  //makes word lower case
   word = word.toLowerCase();
-  console.log('translating: ' + word);
+  console.log('lowercasing: ' + word);
   var vowelIndex = -1;
+  //stores first part of word before vowel
   var firstPart = '';
+  //stores vowel and rest of word
   var restWord = '';
+  //stores only the vowel
   var vowel = '';
+  //checks to see if it starts with a vowel and if it does it adds 'yay' to the end
   if (word.slice(0, 1) === 'a' || word.slice(0, 1) === 'e' || word.slice(0, 1) === 'i' || word.slice(0, 1) === 'o' || word.slice(0, 1) === 'u' || word.slice(0, 1) === 'y') {
     return word + 'yay';
   }
```

Comment:
This is a great solution for finding the vowels. Much better than having individual if statements for each vowel.